dashboard/src/components/Banner.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import 'bootstrap/dist/css/bootstrap.min.css';
+import "../styles/main.css";
+import { Nav, Navbar, Button, Form, NavDropdown, FormControl } from "react-bootstrap";
+
+const ROUTES = ["/", "/howto", "/leaderboard", "https://github.com/AndrooTheChen/REMi"];
+const TEXTS = ["Home", "How To", "Leaderboard", "Repository"];
```

Comment:
since you're using ROUTES all across the app, recommend extracting these out into a seperate routes file, maybe in a constants folder. Something like this https://github.com/pennlabs/penn-clubs/blob/master/frontend/constants/routes.ts